pkg/investigations/chgm/chgm.go

```diff
@@ -77,8 +85,23 @@ func InvestigateTriggered(r *investigation.Resources) error {
 	switch verifierResult {
 	case networkverifier.Failure:
 		logging.Infof("Network verifier reported failure: %s", failureReason)
-		metrics.Inc(metrics.ServicelogPrepared, r.AlertType.String(), r.PdClient.GetEventType())
-		notesSb.WriteString(fmt.Sprintf("⚠️ NetworkVerifier found unreachable targets. \n \n Verify and send service log if necessary: \n osdctl servicelog post %s -t https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/required_network_egresses_are_blocked.json -p URLS=%s\n", r.Cluster.ID(), failureReason))
+
+		egressSL.Description = fmt.Sprintf(egressSL.Description, failureReason)
+		err = r.OcmClient.PostServiceLog(r.Cluster.ID(), &egressSL)
+		if err != nil {
+			logging.Error("Failed to send network verifier servicelog: %w", err)
+			notesSb.WriteString(fmt.Sprintf("⚠️ NetworkVerifier found unreachable targets. \n \n Verify and send service log if necessary: \n osdctl servicelog post %s -t https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/required_network_egresses_are_blocked.json -p URLS=%s\n", r.Cluster.ID(), failureReason))
+			break
+		}
+		metrics.Inc(metrics.ServicelogSent, r.AlertType.String(), r.PdClient.GetEventType())
+		notesSb.WriteString(fmt.Sprintf("⚠️ NetworkVerifier found unreachable targets. \n %s\n The servicelog has been sent.\n", failureReason))
+
+		if strings.Contains(failureReason, "nosnch.in") {
+			logging.Info("Alert is expected as the cluster is not able to reach deadmanssnitch. Silencing.")
+			notesSb.WriteString("Silencing the alert, as the cluster is not able to reach deadmanssnitch.")
+			return r.PdClient.SilenceAlertWithNote(notesSb.String())
+		}
+		notesSb.WriteString("Deadmanssnitch is not blocked. Please investigate the cluster.")
```

Comment:
I think this might be confusing for the engineer receiving the alert with the note: the text will be really long and the `Deadmanssnitch is not blocked. Please investigate the cluster` will be somewhere after the long list of urls.

What do you think about these three options...
**option 1: add urls at the end** - do you think this would make it clear that it is unrelated to the alert? 
```
🤖 Automated CHGM pre-investigation 🤖
===========================
✅ Customer did not stop nodes.
✅ Deadmanssnitch URL is not blocked. 
🤖 NetworkVerifier sent a service log for unreachable targets: url1,url2,url3,url4
```
**option 2: leave out urls to not confuse SRE**
```
🤖 Automated CHGM pre-investigation 🤖
===========================
✅ Customer did not stop nodes.
✅ Deadmanssnitch URL is not blocked. 
```
**option 3: let SRE post the SL in case DMS isn't blocked**
This would be the current option: automated SLs only for DMS blocked. It would still allow the SRE to send or not send if they don't wish to.

Reply:
Its a good point. I think I will go for option one, but also a note saying please investigate? 
Otherwise it may be confusing if you do a cluster context and see a network SL has just been sent for the cluster. wdyt?

Reply:
testing here btw https://redhat.pagerduty.com/incidents/Q39EWN72NX2QJH/timeline

Reply:
```
🤖 Automated CHGM pre-investigation 🤖
===========================
✅ Customer did not stop nodes.
⚠️ NetworkVerifier found unreachable targets and sent the SL, but deadmanssnitch is not blocked! 
⚠️ Please investigate this cluster.
Unreachable: 
events.pagerduty.com:443
```